src/main/java/seedu/mtracker/ui/TextUi.java

```diff
@@ -121,7 +121,7 @@ public static void showErrorMessage(Exception e) {
     public static void displayExitMessage() {
         System.out.println(BYE_LOGO);
         System.out.println("Thank you for using mTracker.\n"
-                + "☻ MAY THE MARKETS BE WITH YOU!!! ᕦ(ò_óˇ)ᕤ");
+                + "☺ ☺ ☺ MAY THE MARKETS BE WITH YOU!!! ☺ ☺ ☺");

```

Comment:
Sry maybe I shouldnt have said smiley faces. Just remove the emojis for the exit and greet message. Even the smiley emoji gets corrupted in the display

Reply:
Oh okay sure np